pkg/odo/cli/component/create.go

```diff
@@ -899,7 +914,8 @@ func NewCmdCreate(name, fullName string) *cobra.Command {
 	componentCreateCmd.Flags().StringSliceVar(&co.componentEnvVars, "env", []string{}, "Environmental variables for the component. For example --env VariableName=Value")
 
 	if experimental.IsExperimentalModeEnabled() {
-		componentCreateCmd.Flags().BoolVar(&co.devfileMetadata.downloadSource, "downloadSource", false, "Download sample project from devfile. (ex. odo component create <component_type> [component_name] --downloadSource")
+		componentCreateCmd.Flags().StringVar(&co.devfileMetadata.downloadSource, "downloadSource", "", "Download sample project from devfile. (ex. odo component create <component_type> [component_name] --downloadSource/--downloadSource=<devfile-project>")
```

Comment:
There is a special section for command examples. See https://github.com/openshift/odo/blob/ba8f12df0232a6f12b4206c5211651935af838ad/pkg/odo/cli/component/create.go#L100

Can you put the example showing how to use the flag there? Having it in a flag description makes it really long and looks quite ugly and overwhelming in the terminal.